flink-runtime-web/web-dashboard/app/partials/overview.jade

```diff
@@ -22,6 +22,12 @@ nav.navbar.navbar-default.navbar-fixed-top.navbar-main
   .navbar-title
     | Overview
 
+  .navbar-info.last.first
+    | Version: {{overview['flink-version']}}
+
+  .navbar-info.last.first(ng-if="overview['flink-commit']")
+    | Commit: {{overview['flink-commit']}}
```

Comment:
Will this print "unknown" as commit ID (because the JSON field is not present) or will it be skipped?